packages/iris-grid/src/GotoRow.tsx

```diff
@@ -91,13 +90,18 @@ function GotoRow({
       e.stopPropagation();
       e.preventDefault();
       onGotoValueSubmit();
+    } else if (
+      (e.key === 'Backspace' || e.key === 'Delete') &&
+      (gotoValue === `${Number.POSITIVE_INFINITY}` ||
+        gotoValue === `${Number.NEGATIVE_INFINITY}`)
+    ) {
+      onGotoValueInputChanged('');
     }
   };
 
   const index = model.getColumnIndexByName(gotoValueSelectedColumnName);
 
-  assertNotNull(index);
-  const selectedColumn = columns[index];
+  const selectedColumn = columns[index ?? 0];
```

Comment:
This seems incorrect. Why default to a column if the name isn't found? How did the user get to this state where the name isn't a valid column name?

Unless it's for handling a case where there is no selected column and the user brings up the dialog. In which case, add a comment that the dialog defaults to 1st column if there is no user selection

Reply:
See #1100 , go to always defaults the the first column, so when the first column disappears somehow, the `gotoValueSelectedColumn` is not updated automatically, so the assertion throws an error. Solution is to again default the selectedColumn to index 0.